### PR TITLE
Update Helm docs for consul-k8s 1.2.1

### PR DESCRIPTION
**Pull request by missylbytes**
### Description
Updates helm values docs to be consistent with consul-k8s 1.2.1

### CODE REVIEW

File: website/content/docs/k8s/helm.mdx [hunk 1]
```diff
@@ -288,6 +288,8 @@ Use these links to navigate to a particular top-level stanza.
     - `secretKey` ((#v-global-gossipencryption-secretkey)) (`string: ""`) - The key within the Kubernetes secret or Vault secret key that holds the gossip
       encryption key.
 
+    - `logLevel` ((#v-global-gossipencryption-loglevel)) (`string: ""`) - Override global log verbosity level for gossip-encryption-autogenerate-job pods. One of "trace", "debug", "info", "warn", or "error".
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:291]:
```suggestion
    - `logLevel` ((#v-global-gossipencryption-loglevel)) (`string: ""`) - Override global log verbosity level for `gossip-encryption-autogenerate-job` pods. One of "trace", "debug", "info", "warn", or "error".
```

File: website/content/docs/k8s/helm.mdx [hunk 2]
```diff
@@ -361,6 +365,15 @@ Use these links to navigate to a particular top-level stanza.
 
       - `secretKey` ((#v-global-tls-cakey-secretkey)) (`string: null`) - The key within the Kubernetes or Vault secret that holds the CA key.
 
+    - `annotations` ((#v-global-tls-annotations)) (`string: null`) - This value defines additional annotations for
+      tls init jobs. This should be formatted as a multi-line string.
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:369]: active voice

```suggestion
    - `annotations` ((#v-global-tls-annotations)) (`string: null`) - This value defines additional annotations for
      tls init jobs. Format this value as a multi-line string.
```

File: website/content/docs/k8s/helm.mdx [hunk 3]
```diff
@@ -447,6 +462,15 @@ Use these links to navigate to a particular top-level stanza.
         beta.kubernetes.io/arch: amd64
       ```
 
+    - `annotations` ((#v-global-acls-annotations)) (`string: null`) - This value defines additional annotations for
+      acl init jobs. This should be formatted as a multi-line string.
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:466]:
```suggestion
    - `annotations` ((#v-global-acls-annotations)) (`string: null`) - This value defines additional annotations for
      acl init jobs. Format this value as a multi-line string.
```

File: website/content/docs/k8s/helm.mdx [hunk 4]
```diff
@@ -500,6 +524,8 @@ Use these links to navigate to a particular top-level stanza.
         -o jsonpath="{.clusters[?(@.name=='<your cluster name>')].cluster.server}"
       ```
 
+    - `logLevel` ((#v-global-federation-loglevel)) (`string: ""`) - Override global log verbosity level for the create-federation-secret-job pods. One of "trace", "debug", "info", "warn", or "error".
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:527]:
```suggestion
    - `logLevel` ((#v-global-federation-loglevel)) (`string: ""`) - Override global log verbosity level for the `create-federation-secret-job` pods. One of "trace", "debug", "info", "warn", or "error".
```

File: website/content/docs/k8s/helm.mdx [hunk 5]
```diff
@@ -1758,6 +1792,14 @@ Use these links to navigate to a particular top-level stanza.
 
         - `minInstances` ((#v-connectinject-apigateway-managedgatewayclass-deployment-mininstances)) (`integer: 1`)
 
+      - `openshiftSCCName` ((#v-connectinject-apigateway-managedgatewayclass-openshiftsccname)) (`string: restricted-v2`) - The name of the OpenShift SecurityContextConstraints resource to use for Gateways.
+        Only applicable if `global.openshift.enabled` is true.
+
+      - `mapPrivilegedContainerPorts` ((#v-connectinject-apigateway-managedgatewayclass-mapprivilegedcontainerports)) (`integer: 0`) - This value defines the amount we will add to privileged container ports on gateways that use this class.
+        This is useful if you don't want to give your containers extra permissions to run privileged ports.
+        Example: The gateway listener is defined on port 80, but the underlying value of the port on the container
+        will be the 80 + the number defined below.
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:1801]:
```suggestion
      - `mapPrivilegedContainerPorts` ((#v-connectinject-apigateway-managedgatewayclass-mapprivilegedcontainerports)) (`integer: 0`) - This value defines the amount Consul will add to privileged container ports on gateways that use this class.
        This is useful if you don't want to give your containers extra permissions to run privileged ports.
        Example: The gateway listener is defined on port 80, but the underlying value of the port on the container
        will be the 80 + the number defined below.
```

File: website/content/docs/k8s/helm.mdx [hunk 6]
```diff
@@ -2122,10 +2164,12 @@ Use these links to navigate to a particular top-level stanza.
 - `meshGateway` ((#v-meshgateway)) - [Mesh Gateways](/consul/docs/connect/gateways/mesh-gateway) enable Consul Connect to work across Consul datacenters.
 
   - `enabled` ((#v-meshgateway-enabled)) (`boolean: false`) - If [mesh gateways](/consul/docs/connect/gateways/mesh-gateway) are enabled, a Deployment will be created that runs
-    gateways and Consul Connect will be configured to use gateways.
+    gateways and Consul service mesh will be configured to use gateways.
     This setting is required for [Cluster Peering](/consul/docs/connect/cluster-peering/k8s).
     Requirements: consul 1.6.0+ if using `global.acls.manageSystemACLs``.
 
+  - `logLevel` ((#v-meshgateway-loglevel)) (`string: ""`) - Override global log verbosity level for mesh-gateway-deployment pods. One of "trace", "debug", "info", "warn", or "error".
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:2171]:
```suggestion
  - `logLevel` ((#v-meshgateway-loglevel)) (`string: ""`) - Override global log verbosity level for `mesh-gateway-deployment` pods. One of "trace", "debug", "info", "warn", or "error".
```

File: website/content/docs/k8s/helm.mdx [hunk 7]
```diff
@@ -2289,6 +2333,8 @@ Use these links to navigate to a particular top-level stanza.
 
   - `enabled` ((#v-ingressgateways-enabled)) (`boolean: false`) - Enable ingress gateway deployment. Requires `connectInject.enabled=true`.
 
+  - `logLevel` ((#v-ingressgateways-loglevel)) (`string: ""`) - Override global log verbosity level for ingress-gateways-deployment pods. One of "trace", "debug", "info", "warn", or "error".
```
Comment (im2nguyen) [inline website/content/docs/k8s/helm.mdx:2336]:
```suggestion
  - `logLevel` ((#v-ingressgateways-loglevel)) (`string: ""`) - Override global log verbosity level for `ingress-gateways-deployment` pods. One of "trace", "debug", "info", "warn", or "error".
```

**Review by im2nguyen — APPROVED** [review]

**Comment by missylbytes** [comment]
Removed backport label @im2nguyen since this is a PR directly into the release/1.16.x branch

**Pull request merged**